hadoop-build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -174,6 +174,7 @@
         <module name="EmptyStatement"/>
         <module name="EqualsHashCode"/>
         <module name="HiddenField">
+          <property name="ignoreFormat" value="init"/>
```

Comment:
Let's not do this.
It has lots of side effects.

Reply:
Thanks for your suggestion, I will restore this part of the configuration.